sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -85,107 +63,141 @@ extensions:
+        # this part stolen from util.sh configure_os_server()
+        sudo cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.orig.yaml
+        ocver=$( oc version | awk -F'[ v.-]+' '/^oc / {print $2 $3 $4}' )
+        patchcmd="sudo oc ex config patch"
+        if [ $ocver -lt 380 ] ; then
+            patchcmd="docker exec origin openshift ex config patch"
+        fi
+        $patchcmd ${SERVER_CONFIG_DIR}/master/master-config.orig.yaml \
+            --patch="{\"assetConfig\": {\"loggingPublicURL\": \"https://kibana.${public_ip}.xip.io\"}}" | \
+            sudo tee ${SERVER_CONFIG_DIR}/master/master-config.yaml > /dev/null
+        sudo cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.save.yaml
+        $patchcmd ${SERVER_CONFIG_DIR}/master/master-config.save.yaml \
+            --patch="{\"networkConfig\": {\"externalIPNetworkCIDRs\": [\"0.0.0.0/0\"]}}" | \
+            sudo tee ${SERVER_CONFIG_DIR}/master/master-config.yaml > /dev/null
+        # restart cluster so changes will take effect
+        sudo oc cluster down
+        # have to restart docker while oc cluster is down, after it has been initialized, or inter-pod
+        # networking does not work
+        sudo systemctl restart docker
+        sudo oc cluster up --use-existing-config --public-hostname="${public_hostname}" --routing-suffix="${public_ip}.xip.io"
     - type: "script"
       title: "expose the kubeconfig"
       script: |-
+        if [ ! -d $HOME/.kube ] ; then
+          mkdir $HOME/.kube
+        fi
+        sudo cp /var/lib/origin/openshift.local.config/master/admin.kubeconfig $HOME/.kube/config
+        sudo chown $USER $HOME/.kube/config
+        sudo mkdir -p /etc/origin/master
+        sudo cp /var/lib/origin/openshift.local.config/master/admin.kubeconfig /etc/origin/master/admin.kubeconfig
         sudo chmod a+x /etc/ /etc/origin/ /etc/origin/master/
         sudo chmod a+rw /etc/origin/master/admin.kubeconfig
-    - type: "script"
-      title: "ensure built version of origin is installed"
-      timeout: 600
-      repository: "origin"
-      script: |-
-        pushd "/data/src/github.com/openshift/origin-aggregated-logging/"
-        curbranch=$( git rev-parse --abbrev-ref HEAD )
-        popd
-        if [[ "${curbranch}" == master ]] ; then
-           origin_package="$( source hack/lib/init.sh; os::build::rpm::format_nvra )"
-           rpm -V "${origin_package}"
-        fi
     - type: "script"
       title: "install origin-aggregated-logging"
-      repository: "aos-cd-jobs"
+      repository: "openshift-ansible"
       script: |-
-        playbook_base='/usr/share/ansible/openshift-ansible/playbooks/'
+        OS_A_C_J_DIR=/data/src/github.com/openshift/aos-cd-jobs
+        pushd $OS_A_C_J_DIR > /dev/null
+        # cannot use the networking related parameters with cluster up deployment
```

Comment:
Anything that cannot be used for all deployments should not be in this file. Move those options out of the file and into `-e` statements for the jobs that need them.

Reply:
Then it should be removed from the logging.yaml inventory

Reply:
100% -- the intent of those files was that they contain super basic options _everyone_ was going to use. If that's not the case, then we should make sure it is.

Reply:
hmm - these parameters are in the general.yaml, which is used by _all_ jobs. If I remove those parameters from general.yaml, I'll have to update _every_ job.